node/nodeRunner.go

```diff
@@ -113,32 +113,29 @@ func (nr *nodeRunner) Start() error {
 }
 
 func printEnableEpochs(configs *config.Configs) {
```

Comment:
It's nice that we have this function but it has one disadvantage:
we are only testing that the configs were loaded successfully from the config file not that those flags were correctly passed to inner processors. I suggest to keep this function though but rephrasing the prints so that all of them will start with "read enable epoch for xxx" and to add log prints in each and every constructor that use these configs that will resemble to something like "enable epoch for xxx". We will easily test that you correctly passed all arguments just by checking the logs with a grep command on "enable epoch" (all values should appear at least twice with the same vale)